src/DotVVM.Framework/Binding/CustomDefaultProperty.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DotVVM.Framework.Controls;
+
+namespace DotVVM.Framework.Binding
+{
+    public class CustomDefaultProperty : DotvvmProperty
```

Comment:
When I see this name I can't imagine what is it good for. Is `DotvvmPropertyWithFallback` better?